From 3f3074e473b0e34b4698b1cd4df1618b6d51b038 Mon Sep 17 00:00:00 2001
From: Ming Ma <mingma@apache.org>
Date: Fri, 23 Oct 2015 08:36:33 -0700
Subject: [PATCH 2089/2748] YARN-2913 DOCS. Fair scheduler should have ability
 to set MaxResourceDefault for each queue. (Siqi
 Li via mingma)

(cherry picked from commit 934d96a334598fcf0e5aba2043ff539469025f69)
(cherry picked from commit 4bb7e68eb6b601b424b46b1d0e8c92767959c733)

Change-Id: I323fd41f11213b2b404c1e2366d723297d19258b
---
 .../src/site/apt/FairScheduler.apt.vm              |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/FairScheduler.apt.vm b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/FairScheduler.apt.vm
index e25fcbe..6a203b2 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/FairScheduler.apt.vm
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/apt/FairScheduler.apt.vm
@@ -311,6 +311,9 @@ Allocation file format
  * <<A queueMaxAppsDefault element>>, which sets the default running app limit
    for queues; overriden by maxRunningApps element in each queue.
 
+ * <<A queueMaxResourcesDefault element>>: which sets the default max resource limit
+   for queue; overriden by maxResources element in each queue.
+
  * <<A queueMaxAMShareDefault element>>, which sets the default AM resource
    limit for queue; overriden by maxAMShare element in each queue.
 
@@ -381,6 +384,7 @@ Allocation file format
   </queue>
 
   <queueMaxAMShareDefault>0.5</queueMaxAMShareDefault>
+  <queueMaxResourcesDefault>40000 mb,0vcores</queueMaxResourcesDefault>
 
   <!-- Queue 'secondary_group_queue' is a parent queue and may have
        user queues under it -->
-- 
1.7.9.5

